tools/os_deps/deps.yum.RHEL8

```diff
@@ -6,8 +6,10 @@ firefox
 gcc-c++
 git
 jq
+libcurl-devel
```

Comment:
@santos1709 , it turns out that most of our dynamic agents use RHEL7, so you will need to add these two lines to the deps.yum.RHEL7 file as well.